libs/barista-components/filter-field/src/filter-field.ts

```diff
@@ -169,8 +168,6 @@ export class DtFilterFieldCurrentFilterChangeEvent<T> {
   ) {}
 }
 
-export const DT_FILTER_FIELD_TYPING_DEBOUNCE = 200;
```

Comment:
Alright, this seems a bit drastic but I get where the change is coming from. 
If it still works and does not produce side effects. After all we are emitting way `inputChange` outputs now than before. There probably are cases where consumers bind to this event and go off an fetch async data.

This would be a good hint to include into the changelog, so people are aware.